10-setup-htcondor.sh

```diff
@@ -346,6 +348,23 @@ MASTER_ATTRS = \$(MASTER_ATTRS) AnnexName ACCEPT_JOBS_FOR_HOURS ACCEPT_IDLE_MINU
 
 # policy
 use policy : Hold_If_Memory_Exceeded
+
+STARTD_CRON_JOBLIST = $(STARTD_CRON_JOBLIST) base userenv
+
+STARTD_CRON_base_EXECUTABLE = /usr/sbin/osgvo-advertise-base
+STARTD_CRON_base_PERIOD = 4m
+STARTD_CRON_base_MODE = periodic
+STARTD_CRON_base_RECONFIG = true
+STARTD_CRON_base_KILL = true
+STARTD_CRON_base_ARGS =
+
+STARTD_CRON_userenv_EXECUTABLE = $LOCAL_DIR/main/singularity_wrapper.sh
```

Comment:
```suggestion
STARTD_CRON_userenv_EXECUTABLE = $LOCAL_DIR/main/osgvo-advertise-userenv
```
no?

Reply:
No, it has to run via the singularity wrapper. The userenv is adverting the environment inside the default image.

Reply:
I see.